### ISSUE TITLE
Warning when sending Set-Cookie with negative Max-Age

### ISSUE THREAD

**Issue by richdougherty** [playframework/playframework#8205]
This is a warning with the Akka HTTP backend. This warning will go away once we update Akka HTTP and enable a new setting. See: https://github.com/akka/akka-http/issues/1550

```
2018-01-31T06:29:53.538Z [warn] akka.actor.ActorSystemImpl [sourceThread=application-akka.actor.default-dispatcher-42, akkaSource=akka.actor.ActorSystemImpl(application), sourceActorSystem=application, akkaTimestamp=06:29:53.537UTC] - Illegal header: Illegal 'set-cookie' header: Invalid input '-', expected OWS or longNumberCappedAtIntMaxValue (line 1, column 22): PLAY_FLASH=; Max-Age=-86400; Expires=Tue, 30 Jan 2018 06:29:53 GMT; Path=/; HTTPOnly
```

However, we should probably check what the 'correct' behaviour for Max-Age is. Perhaps it should be `Max-Age=0`? Here are some links that @gmethvin found. We might need to do some cross-browser testing if we change the value that Play sends so that we can be sure it works in major browsers.

- https://stackoverflow.com/questions/15932957/difference-between-0-and-negative-value-for-setmaxage-for-cookie
- https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie

**Comment by richdougherty** [comment]
See also: #7680

**Comment by marcospereira** [comment]
Closed by #8314.
